nvchecker/archlinux-proaudio.toml

```diff
@@ -29,6 +29,12 @@ use_max_tag = true
 include_regex = '^v[\.\d]+$'
 prefix = "v"
 
+[airwindows-lv2]
+source = "github"
+github = "hannesbraun/airwindows-lv2"
```

Comment:
I chose the repo on Sourcehut because the upstream author states that (my emphasis)

> I'll still keep the GitHub mirror up-to-date *for some time* and you can still open issues and pull requests on GitHub, if you prefer that.

I've updated the nvchecker config.